paper/paper.md

```diff
@@ -34,110 +34,98 @@ bibliography: bibliography.bib
 
-The functionality in this package integrates into Julia's [`SciML`](https://sciml.ai/) ecosystem. The model is provided in symbolic form as an `ODESystem` using `ModelingToolkit.jl`[@ma2021modelingtoolkit] with additional frequency information for the observed and control variables. Both ODE and DAE system can be provided. `DynamicOED.jl` augments the given system symbolically with the its sensitivty equations and the dynamics of the FIM. The resulting system together with a sufficient information criterion defines an `OEDProblem`. Here, all sampling and control decisions are discretized in time and can be used to model additional constraints. At last, the `OEDProblem` can be transformed into an `OptimizationProblem` as a sufficient input to `Optimization.jl` [@vaibhav_kumar_dixit_2023_7738525]. Here, a variety of optimization solvers for nonlinear programming and mixed-integer nonlinear programming available as additional backends, e.g. `Ipopt` [@Waechter2006]. A simple example demonstrates the usage of `DynamicOED.jl` for the Lotka-Volterra system [@Sager2013]. 
 
-\pagebreak
+\autoref{fig:lotka} shows the solution of the example above including the differential states, sensitivities $G$ and the sampling decisions $w$. More examples can be found in the [documentation](https://mathopt.github.io/DynamicOED.jl/dev/). 
+
+\newpage
 
 ```julia
 using DynamicOED
 using ModelingToolkit
 using Optimization, OptimizationMOI, Ipopt
 
 @variables t
-@variables x(t)=0.5 [description = "Biomass Prey"] y(t)=0.7 [description ="Biomass Predator"]
-@variables u(t) [description = "Control"]
-@parameters p[1:2]=[1.0; 1.0] [description = "Fixed Parameters", tunable = false]
-@parameters p_est[1:2]=[1.0; 1.0] [description = "Tunable Parameters", tunable = true]
+@variables x(t)=0.5 [description="Biomass Prey"] 
+@variables y(t)=0.7 [description="Biomass Predator"]
+@variables u(t) [description="Control"]
+@parameters p[1:2]=[1.0;1.0] [description="Fixed Parameters", tunable=false]
+@parameters p_est[1:2]=[1.0;1.0] [description="Tunable Parameters", tunable=true]
 D = Differential(t)
-@variables obs(t)[1:2] [description = "Observed", measurement_rate = 96]
+@variables obs(t)[1:2] [description = "Observed", measurement_rate=96]
 obs = collect(obs)
 
-# Define the system
 @named lotka_volterra = ODESystem(
     [
         D(x) ~   p[1]*x - p_est[1]*x*y;
         D(y) ~  -p[2]*y + p_est[2]*x*y
     ], tspan = (0.0, 12.0),
     observed = obs .~ [x; y]
 )
-
-# Extend the system
 @named oed_system = OEDSystem(lotka_volterra)
-
-# Define a OED problem 
 oed_problem = OEDProblem(structural_simplify(oed_system), DCriterion())
 
-# Define constraints 
 optimization_variables = states(oed_problem)
 
 constraint_equations = [
-    sum(optimization_variables.measurements.w₁) ≲ 32,
-    sum(optimization_variables.measurements.w₂) ≲ 32,
+    sum(optimization_variables.measurements[1]) ≲ 32,
```

Comment:
# Collect the variable containers
w1, w2 = collect(valkeys(optimization_variables.measurements))

# Define constraint equations
constraint_equations = [
    sum(optimization_variables.measurements[w1]) \leq 32, 
    sum(optimization_variables.measurements[w2]) \leq 32, 
]

Reply:
w1, w2 = keys(optimization_variables.measurements)